src/wazuh_db/wdb.c

```diff
@@ -1049,6 +1049,52 @@ cJSON* wdb_exec_stmt_sized(sqlite3_stmt * stmt, const size_t max_size, int* stat
     return result;
 }
 
+int wdb_exec_stmt_send(sqlite3_stmt* stmt, int peer) {
+    if (!stmt) {
+        mdebug1("Invalid SQL statement.");
+        return OS_INVALID;
+    }
+    if (OS_SetSendTimeout(peer, WDB_BLOCK_SEND_TIMEOUT_S) < 0) {
+        merror("Socket %d error setting timeout: %s (%d)", peer, strerror(errno), errno);
+        return OS_SOCKTERR;
+    }
+
+    int status = OS_SUCCESS;
+    int sql_status = SQLITE_ERROR;
+    cJSON * row = NULL;
+    char* response = NULL;
+    char* header = "due ";
+    int header_size = strlen(header);
+    os_calloc(OS_MAXSTR, sizeof(char), response);
+    memcpy(response, header, header_size);
+    char* payload = response+header_size;
+    int payload_size = OS_MAXSTR-header_size;
```

Comment:
An extra space should be added before and after "+" and "-"

Reply:
The header trick is really good!
But I think these lines take a time to be understood. Maybe, we could add some comments.

Reply:
Done!